External/FEXCore/Source/Interface/Core/JIT/Arm64/ALUOps.cpp

```diff
@@ -84,6 +84,21 @@ DEF_OP(Add) {
   }
 }
 
+DEF_OP(TestNZ) {
+  auto Op = IROp->C<IR::IROp_TestNZ>();
+  const uint8_t OpSize = Op->Size;
+
+  LOGMAN_THROW_AA_FMT(OpSize == 4 || OpSize == 8, "Unsupported {} size: {}", __func__, OpSize);
+  const auto EmitSize = OpSize == 8 ? ARMEmitter::Size::i64Bit : ARMEmitter::Size::i32Bit;
+
+  const auto Dst = GetReg(Node);
+  const auto ZeroReg = ARMEmitter::Reg::zr;
+  adds(EmitSize, ZeroReg, GetReg(Op->Src1.ID()), ZeroReg);
```

Comment:
We should add the `adds` alias for `cmn` in to our emitter.
I missed a bunch of aliases when writing it.